src/test/java/hudson/plugins/git/UserRemoteConfigTest.java

```diff
@@ -29,16 +36,17 @@
 
     @Issue("JENKINS-38048")
     @Test
+    @Ignore
     public void credentialsDropdown() throws Exception {
         SystemCredentialsProvider.getInstance().getCredentials().add(new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, "mycreds", null, "jenkins", "s3cr3t"));
         SystemCredentialsProvider.getInstance().save();
         FreeStyleProject p1 = r.createFreeStyleProject("p1");
         FreeStyleProject p2 = r.createFreeStyleProject("p2");
         r.jenkins.setSecurityRealm(r.createDummySecurityRealm());
         r.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy().
-            grant(Jenkins.ADMINISTER).everywhere().to("admin").
-            grant(Jenkins.READ, Item.READ).everywhere().to("dev").
-            grant(Item.EXTENDED_READ).onItems(p1).to("dev"));
+                grant(Jenkins.ADMINISTER).everywhere().to("admin").
```

Comment:
My bad. Apologies for those whitespace.(i had a different formatting). I will keep that in mind.